backend-docker/tests/apiserver_test.go

```diff
@@ -0,0 +1,106 @@
+package tests
+
+import (
+	"testing"
+	"github.com/EUDAT-GEF/GEF/backend-docker/config"
+	"github.com/EUDAT-GEF/GEF/backend-docker/server"
+	"net/http/httptest"
+	"net/http"
+	"encoding/json"
+	"log"
+	"io/ioutil"
+)
+
+func isJSON(s string) bool {
+	var js map[string]interface{}
+	return json.Unmarshal([]byte(s), &js) == nil
+}
+
+func TestServer(t *testing.T) {
+	settings, err := config.ReadConfigFile(configFilePath)
+	if err != nil {
+		t.Error("FATAL while reading config files: ", err)
+	}
+	clientConf = settings.Docker
+
+	c := newClient(t)
+	s := server.NewServer(settings.Server, c)
+
+	srv := httptest.NewServer(s.Server.Handler)
+	baseURL := srv.URL + "/api/"
+
+	ifAPIExist(baseURL, t)
+	callListVolumesHandler(baseURL + "volumes", t)
+}
+
+func ifAPIExist(callURL string, t *testing.T) bool {
```

Comment:
Recommendation: it's better to start function names with a verb. I would name this `checkIfAPIExists`